spec/index.bs

```diff
@@ -488,7 +488,7 @@ dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
 ## Buffer Usage ## {#buffer-usage}
 
 <script type=idl>
-typedef GPUFlags GPUBufferUsageFlags;
+typedef unsigned long GPUBufferUsageFlags;
 interface GPUBufferUsage {
     const GPUFlags NONE      = 0x0000;
```

Comment:
GPUFlags should be removed, and all of these should use GPUBufferUsageFlags. Similar for the other usages.

Reply:
Done.